docs/en/drivers/safari.md

```diff
@@ -0,0 +1,47 @@
+## The Safari Driver
+
+Safari driver has been added to Appium since version 1.20. This driver
+is a wrapper over Apple's [safaridriver](https://developer.apple.com/documentation/webkit/testing_with_webdriver_in_safari?language=objc)
+binary, which is included to the standard Mac OS distribution and implements communication with either
```

Comment:
`macOS` is proper than `Mac OS` in newer versions. (probably Since Sierra??)

Reply:
changed